Mathlib/Order/WellFounded.lean

```diff
@@ -297,3 +297,12 @@ noncomputable def WellFoundedGT.toOrderTop {α} [LinearOrder α] [Nonempty α] [
     OrderTop α :=
   have := WellFoundedLT.toOrderBot (α := αᵒᵈ)
   inferInstanceAs (OrderTop αᵒᵈᵒᵈ)
+
+instance WellFoundedLT.ulift [LT α] [WellFoundedLT α] : WellFoundedLT (ULift α) where
+  wf := by
+    refine ⟨fun ⟨x⟩ ↦ ?_⟩
```

Comment:
Is there some API like `WellFounded.invImage` that you can use more directly here, noting that `a < b` is really just `a.down < b.down`?

Reply:
Ah, `InvImage.wf`! Probably that should be renamed for dot notation, but that's a problem for another PR.